app/src/main/java/cm/aptoide/pt/app/view/GridAppDisplayable.java

```diff
@@ -32,7 +31,6 @@ public GridAppDisplayable(App pojo, String tag, boolean totalDownloads,
       NavigationTracker navigationTracker, StoreContext storeContext) {
     super(pojo);
     this.tag = tag;
-    this.totalDownloads = totalDownloads;
```

Comment:
You forgot to remove it from the constructor

Reply:
done